features/interactive_comment.feature

```diff
@@ -6,4 +6,17 @@ Scénario: d'une vidéo
   Soit "Vidéo Sherlock Jr. (Buster Keaton)" le document principal
   Et "Note rire Buster Keaton (Antoine-Valentin Charpentier)" une des gloses ouverte
   Quand je clique sur la référence temporelle "00:03:09.000 --> 00:03:15.000" avec pour commentaire "Une feuille de papier est collée sur le balais."
-  Alors la vidéo du document principal se lance de "189" secondes à "195" secondes
\ No newline at end of file
+  Alors la vidéo du document principal se lance de "189" secondes à "195" secondes
+
+Scénario: d'un commentaire
+
+  Soit "Treignes, le 8 septembre 2012 (Christophe Lejeune)" le document principal
+  Et "Etage suivant (Christophe Lejeune)" une des gloses ouverte
+  Quand je passe mon curseur sur le texte :
```

Comment:
"je survole" ?

Reply:
Great suggestion, text is replaced in commit https://github.com/Hypertopic/HyperGlosae/commit/51d41dbb74a7547abdb8a8570e42814da4d3af43